ops/ccs-ops-misc/load_test/common/stats/aggregated_stats.py

```diff
@@ -0,0 +1,191 @@
+"""Members of this file/module should be related to the collection of performance statistics during
+a test run as well as the representation of those statistics via dataclasses or other suitable objects"""
+from dataclasses import dataclass, fields
+from locust.stats import StatsEntry, sort_stats, PERCENTILES_TO_REPORT
+from locust.env import Environment
+import time
+from typing import Any, Dict, List, Optional
+
+from common.stats.stats_config import StatsEnvironment
+
+
+class StatsCollector(object):
+    """Used to collect a snapshot of aggregated performance statistics of all tasks, or endpoints, that
+    ran in the current Locust environment"""
+
+    def __init__(self, locust_env: Environment, stats_tag: str, running_env: StatsEnvironment = StatsEnvironment.TEST) -> None:
+        """Creates a new instance of StatsCollector given the current Locust environment and a list of percentiles to report.
+
+        Args:
+            locust_env (Environment): Current Locust environment
```

Comment:
I had a feeling this would potentially be a bit confusing -- there is Locust's `Environment` class which exposes several fields and methods related to information about the tests running in Locust like the parsed Locust parameters, the number of users spawned, etc. And then there is `StatsEnvironment` which represents the target application environment you're talking about (i.e. `TEST`, `PROD`, etc.).

I can definitely try to make this distinction clearer either through comments or by using naming that's a bit more distinct.

Reply:
Maybe a small comment would be helpful, since I got confused at least. Thanks for the explanation!